### PR TITLE
fix: more tweaks to the readme

### PR DESCRIPTION
**Pull request by acossta**

### CODE REVIEW

**Review by davepropel — COMMENTED** [review]
Looks good. double checked the yarn + npm install commands, working fine.

File: README.md [hunk 1]
```diff
@@ -12,27 +12,47 @@
...
+yarn add @propeldata/react-leaderboard
+```
+
+or `npm`:
+
+```shell
+npm install @propeldata/react-counter
+npm install @propeldata/react-time-series
+npm install @propeldata/react-leaderboard
+```
```
[the remark this answers is not in the document]
Comment (markandrus) [reply README.md:33]: This change could be applied to all the other README.md files in the repo. For example, we have similar install instructions for Counter, Time Series, and Leaderboard.

**Review by markandrus — COMMENTED** [review]
I think "Why the Propel UI Kit?" and "Core principles" take up too much space at the top of the README.md and require too much scrolling to get to an actual example.

Not that this isn't an important question, and not that core principles shouldn't be stated, it's just a lot of real estate compared to other component libraries' README.md files, and these questions can be answered elsewhere. For example, some projects have a DESIGN.md (see [here](https://github.com/google/mundane/blob/master/DESIGN.md) or [here](https://github.com/google/fleetspeak/blob/master/DESIGN.md)) and others have a PRINCIPLES.md (see [here](https://github.com/moby/moby/blob/master/project/PRINCIPLES.md) or [here](https://github.com/jeromeetienne/tquery/blob/master/principles.md)).

Compare to

- [tremorlabs/tremor](https://github.com/tremorlabs/tremor) — README.md includes the value prop (short), usage, minimal example; otherwise, links out to documentation.
- [chakra-ui/chakra-ui](https://github.com/chakra-ui/chakra-ui) — README.md includes the value prop (short), features, usage, minimal example; otherwise, links out to documentation.
- [radix-ui/primitives](https://github.com/radix-ui/primitives) — README.md includes the value prop (short); otherwise, links out to documentation.
- [clauderic/dnd-kit](https://github.com/clauderic/dnd-kit) — README.md includes the value prop (bullet point list), key concepts; otherwise, links out to documentation.

Proposal:

1. Turn "Core principles" and "Key features" into a tight bullet point list and move them under our value prop, like clauderic/dnd-kit.
2. Move "Why the Propel UI Kit?" to DESIGN.md or PRINCIPLES.md and link out to it from README.md.

In the future, I think we should build a Docusaurus site in this repo to host longer-form documentation, with interactable examples and code samples. Most other component libraries seem to quickly outgrow README.md for documenting everything.

File: README.md [hunk 2]
```diff
@@ -96,9 +74,13 @@ You can customize the component using your own styles.
   <img src="https://storybook.propeldata.com/images/time-series.png"/>
 </p>
 
-### Connected Mode: The Power of Propel one import away
+### Connected Mode
+
+In the "Connected" mode, the component automatically fetches the data from the Propel API. You don't need a backend, configure a GraphQL client, or handle loading and error states.
```
Comment (markandrus) [inline README.md:79]:
```suggestion
In the "Connected" mode, the component automatically fetches the data from the Propel API. You don't need a backend, to configure a GraphQL client, or handle loading and error states.
```

File: PRINCIPLES.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+- **Work and look great out-of-box.** The components work and look great out of the box. You can drop them into your app with minor color customization, and you'll be off to a great start.
+
+- **Extremely customizable.** You can customize their styles, look and feel, and error and loading experience to always feel native to your product.
+
+- **Built in React for React.** We built them in React for React frontends. No cross-framework abstractions and no unnecessary baggage.
+
+- **They work with or without Propel's APIs.** You can use them by themselves in Static mode or with the Propel API in Connected mode.
+
+Back to the [README](README.md)
```
Comment (markandrus) [inline PRINCIPLES.md:23]: Maybe in increase the URL to the whole text so it's more clickable?
```suggestion
[Back to the README](README.md)
```

**Review by markandrus — APPROVED** [review]
I think it's great!

**Pull request merged**